components/backends/trtllm/src/dynamo/trtllm/main.py

```diff
@@ -318,7 +318,7 @@ async def init(runtime: DistributedRuntime, config: Config):
             )
 
         # Get health check payload (checks env var and falls back to TensorRT-LLM default)
-        health_check_payload = TrtllmHealthCheckPayload().to_dict()
+        health_check_payload = TrtllmHealthCheckPayload(tokenizer=tokenizer).to_dict()
```

Comment:
Is there no prefill vs. decode specific payloads for trtllm like there are in your vllm/sglang PRs?